server/routerlicious/packages/services-client/src/auth.ts

```diff
@@ -26,11 +26,17 @@ export function generateToken(
         user = generateUser();
     }
 
+    // Current time in seconds
+    const now = Math.round((new Date()).getTime() / 1000);
+
     const claims: ITokenClaims = {
         documentId,
         scopes,
         tenantId,
         user,
+        iat: now,
+        exp: now + 5 * 60, // 5 minute expiration window
+        ver: "1.0",
```

Comment:
I figured these are a good default. Should I add `tokenLifetime` and `version` as parameters to `generateToken`?

Reply:
Yes. Add lifetime and ver as optional params. And use the default when not present. Also default lifetime should be at an hour. Or closest so that the validation step still passes.